src/IO/H5/Python/DeleteSubfile.py

```diff
@@ -0,0 +1,33 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import click
+import h5py
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument("h5files",
+                type=click.Path(exists=True,
+                                file_okay=True,
+                                dir_okay=False,
+                                readable=True),
+                nargs=-1)
+@click.option("--subfile",
+              "-d",
+              "subfiles",
+              required=True,
+              multiple=True,
+              help="Subfile to delete")
+def delete_subfile_command(h5files, subfiles):
```

Comment:
Since you can pass multiple subfiles to be deleted, can you rename everything to `delete_subfiles` rather than `delete_subfile` (CLI name, file name, function name, etc.)? That way it's clear you can delete multiple at once.

Reply:
done, also added missing docstring